packages/data-plugin-persistence/package.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
We probably reach the point where we should create a script which scaffolds new package 😛

Reply:
It's quite prone to copypasta 😄 🍝